### PR TITLE
PKU-170

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Edge case permission fix.  This forces the overview screen check for permissions every time so that it is always up to date with the current state of authorization.  If authorization is denied, they are not allowed to continue past the overview screen.

### CODE REVIEW

**Comment by DanWebster** [comment]
Looks great
IMG_2DDBACA8EC25-1

File: BiomarinPKU/BiomarinPKU-Study/ActivityViewController.swift [hunk 1]
```diff
@@ -261,6 +261,11 @@ class ActivityViewController: UIViewController, RSDTaskViewControllerDelegate {
     }
     
     @IBAction func physicalTapped() {
+        // Work-around fix for permission bug
+        // This will force the overview screen to check permission state every time
+        // Usually research framework caches it and the state becomes invalid
+        UserDefaults.standard.removeObject(forKey: "rsd_MotionAuthorizationStatus")
```
Comment (syoung-smallwisdom) [inline BiomarinPKU/BiomarinPKU-Study/ActivityViewController.swift]: Honestly, I don't think this is a good idea. It can take time to check for authorization every time. Should a user go to the trouble of changing their settings *after* they've previously approved them, then really, is it that onerous for them to have to step through the steps to find out that they can't run the test? I mean, I get it, but since this is an edge case, it seems that the better user experience is to allow them to move quickly past the first overview screen.
